app/assets/stylesheets/style.css.scss

```diff
@@ -1,6 +1,43 @@
 @import url(https://fonts.googleapis.com/css?family=Pacifico);
 @import url(https://fonts.googleapis.com/css?family=Lato:300,400);
 @import 'foundation-icons';
+@import "font-awesome-sprockets";
+@import "font-awesome";
+
+.fa-star-o {
+  font-size: 2em;
+}
+
+.fa-star {
+  font-size: 2em;
+  color: #41A5B2;
+}
+
+.button {
+  font-family: Lato;
+  background-color: #41A5B2;
+}
+.row .row {
+  width: 100%;
+}
+
+.top-bar .left .fi-magnifying-glass {
+  font-size: 2.5em;
+  color: #FFFFFF;
+}
+
+.top-bar .left #search-box .button {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 4